frontend/src/components/BtoDetailsPage.tsx

```diff
@@ -67,7 +70,10 @@ export default function BtoDetailsPage({btoDisplayItems, changeBtoItem, username
                         <ShowDummyText btoDisplayItem={btoItem} username={username}/> : <></>
                     }
                     {btoItem.status === "VOTE" ?
-                        <Vote btoItem={btoItem} username={username} changeBtoItem={changeBtoItem}/>
+                        <div>
+                            <h1>Vote getroffen</h1>
+                            <Vote btoItem={btoItem} username={username} updateVote={updateVote}/>
+                        </div>
                         :
                         <div>
                             <p>AktionOwner: {btoItem.actionOwner}</p>
```

Comment:
See elsewhere, "AktionOwner" could be "Aktionsadmin".

Reply:
Ja, an manchen Stellen kann man noch nachbessern. Im Moment versuche ich zwar die Balance zwischen englischem Programmcode und deutscher Benutzeroberfläche zu schaffen - aber das gelingt noch nicht überall.